tests/Oro/Tests/ORM/AST/Query/Functions/fixtures/mysql/cast.yml

```diff
@@ -81,13 +81,6 @@
   expectedResult:
       - '2014-01-04 05:06:07'
 
-- functions:
```

Comment:
Should be supported
https://dev.mysql.com/doc/refman/8.0/en/json.html

Reply:
Not in MariaDB https://mariadb.com/kb/en/cast/

I reverted the commit and the tests fail

Reply:
MariaDB has never been supported by oro/doctrine-extensions. 
Support for the JSON type is critical and cannot be dropped.

Reply:
Previous CI (Travis) have tests using MariaDB.